tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+	"version": "2.0.3",
+	"compilerOptions": {
+		"declaration": false,
+		"module": "umd",
+		"noImplicitAny": true,
+		"noImplicitThis": true,
```

Comment:
I think there are a couple of other noImplicit* statements we've started to use?